core/storage/feedback/gae_models.py

```diff
@@ -76,6 +76,16 @@ class GeneralFeedbackThreadModel(base_models.BaseModel):
     # The number of messages in the thread.
     message_count = ndb.IntegerProperty(indexed=True, default=0)
 
+    # Details about the last message in the thread.
+    last_message_id = ndb.IntegerProperty(indexed=False, default=None)
```

Comment:
I think it's safe to make all message and author IDs indexed=True. I think you can also skip "default=None" since that's GAE's default.